spark/spark-scala-parent/src/main/scala/org/apache/zeppelin/spark/BaseSparkScalaInterpreter.scala

```diff
@@ -95,7 +95,6 @@ abstract class BaseSparkScalaInterpreter(val conf: SparkConf,
         System.setOut(Console.out)
         interpreterOutput.setInterpreterOutput(context.out)
         interpreterOutput.ignoreLeadingNewLinesFromScalaReporter()
-        context.out.clear()
```

Comment:
is there any other cases of `context.out.clear()`?

Reply:
I move the clear into other places. https://github.com/apache/zeppelin/pull/3341/files#diff-25ff02ea9a7b4a31efaf02e8b401ea5bR191
Otherwise, the deprecated warning message will disappear after the paragraph is finished.

Reply:
ah, but I meant if there might be other calls to clear that will also undo printDeprecateMessage